### DISCUSSION TITLE
Consider disabling stalebot

### DISCUSSION THREAD

**Discussion by abhinav** [yuin/goldmark#359] [General]
Hi @yuin,
This is to discuss the possibility of disabling stalebot on this project.
For a widely used library like this, it doesn't add much value, hides possible bug reports, and discourages contributions.

Multiple legitimate issues have been closed as stale, e.g.:

- https://github.com/yuin/goldmark/issues/228
- https://github.com/yuin/goldmark/issues/227 (albeit this is a feature request so it should be a discussion)
- https://github.com/yuin/goldmark/issues/211

More importantly, multiple PRs have been marked as stale and closed, e.g.:

- https://github.com/yuin/goldmark/pull/336
- https://github.com/yuin/goldmark/pull/326
- https://github.com/yuin/goldmark/pull/317

I want to be clear that I recognize that this is an open source project and you don't owe these issues and PRs your time. I also understand the desire to keep the issue and PR backlog small.
However, I believe that enforcing Stalebot in such a way hides issues that others might have, it discourages contributions, and removes an avenue for community collaboration.

(Some points above borrowed from <https://drewdevault.com/2021/10/26/stalebot.html>.)

So in short: how do you feel about disabling Stalebot?
